src/View/Helper/AbstractTranslatorHelper.php

```diff
@@ -28,7 +28,7 @@
      *
      * @var string
      */
-    protected $translatorTextDomain = 'default';
+    protected $translatorTextDomain = null;
```

Comment:
Mh, the BC break would only be of some subclass overwrote the standard textDomain with "default" again and expecting the new behavior.
Or am I missing something?